cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -74,6 +83,7 @@ public ClusterOperatorConfig(Set<String> namespaces, long reconciliationInterval
      * @param map   map from which loading configuration parameters
      * @return  Cluster Operator configuration instance
      */
+    @SuppressWarnings("checkstyle:NPathComplexity")
```

Comment:
It should be easy to factor out methods for each option to avoid needing this suppression

Reply:
Good point. I should be less lazy :-o. I broke it down to smaller methods.